PowerShell/ScubaGear/baselines/defender.md

```diff
@@ -86,7 +86,7 @@ The standard and strict preset security policies SHALL be enabled.
 
 
 #### MS.DEFENDER.1.2v1
-All users SHALL be added to Exchange Online Protection in either the standard or strict preset security policy.
+All users SHALL be added to Exchange Online Protection (EOP) in either the standard or strict preset security policy.
```

Comment:
is this acronym used in any microsoft documentation?

Reply:
yep, linked below for reference!
https://learn.microsoft.com/en-us/defender-office-365/eop-about